CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ We use *breaking* word for marking changes that are not backward compatible (rel
 
 - [#2502](https://github.com/thanos-io/thanos/pull/2502) Added `hints` field to `SeriesResponse`. Hints in an opaque data structure that can be used to carry additional information from the store and its content is implementation specific.
 - [#2521](https://github.com/thanos-io/thanos/pull/2521) Sidecar: add `thanos_sidecar_reloader_reloads_failed_total`, `thanos_sidecar_reloader_reloads_total`, `thanos_sidecar_reloader_watch_errors_total`, `thanos_sidecar_reloader_watch_events_total` and `thanos_sidecar_reloader_watches` metrics.
+- [#2532](https://github.com/thanos-io/thanos/pull/2532) Store: Added caching bucket, that can cache chunks into shared memcached. This can speed up querying and reduce number of requests to object storage.
```

Comment:
I guess no in-mem option is planned :thinking: 

I think that is a good pattern and simplify a lot.

Reply:
We can surely add it, if it is needed, but it will help only in small installations.

Reply:
Let's skip it , it's good practice with Memcached for now